README.md

```diff
@@ -952,6 +960,10 @@ bazel run --stamp \
   //user_project:exported_lib.publish`
 ```
 
+It's also possible to publish to a Google Cloud Storage bucket (make sure to not add a trailing slash):
```

Comment:
Nit: could we just remove a trailing slash if we see one?

Reply:
Thought about it, but I didn't want to mess with the users args, in case s/he really want the extra slash from some reason.
Now that I think about maybe it doesn't really make sense, will add it.

Reply:
Done.